src/doc/DocDrawingDialog.js

```diff
@@ -249,17 +250,16 @@ class DocDrawingDialog extends AnnotationDialog {
      * the username does not exist.
      *
      * @private
-     * @param {Annotation} savedAnnotation The annotation data to populate the label with.
+     * @param {Annotation} annotation The annotation data to populate the label with.
      * @return {void}
      */
-    assignDrawingLabel(savedAnnotation) {
-        if (!savedAnnotation || !this.drawingDialogEl) {
+    assignDrawingLabel(annotation) {
+        if (!annotation || !this.drawingDialogEl || annotation.user.id === '0') {
```

Comment:
We need to ensure that the user id is not '0' so that we know the API has returned a valid user name and successfully saved the new annotation

Reply:
Are we sure that the annotation will have a user? user.id could throw an error otherwise.

Reply:
The temporary annotation contains a blank user w/ an id of 0. Once the API successfully saves an annotation, it returns a user with a valid id

Reply:
https://github.com/box/box-annotations/blob/master/src/AnnotationService.js#L56